doc/whats_new/v1.0.rst

```diff
@@ -87,6 +87,15 @@ Changelog
   Use ``var_`` instead.
   :pr:`18842` by :user:`Hong Shao Yang <hongshaoyang>`.
 
+- |API|: The parameter ``normalize`` of :class:`linear_model.LinearRegression`
+  is deprecated and will be removed in 1.2. The use of
+  :class:`~sklearn.pipeline.Pipeline` with
+  :class:`~sklearn.preprocessing.StandardScaler` is recommended instead.
```

Comment:
Maybe add a bit more details such as:

The behavior of the deprecated `LinearRegression(normalize=True)`  can be reproduced with
`make_pipeline(StandardScaler(with_mean=False), LinearRegression())`.